### PR TITLE
Add functionality for scanning multiple repos and outputting by contributor csv

### PR DESCRIPTION
**Pull request by jspeed-meyers**
@kdobolyi, if you have time, can you please review this? This is the functionality for building the dataset that will enable testing whether predicting location with commit metadata is feasible and adequately powerful. I realize there is a lot of changed code in this PR. Sorry, probably too big for a single PR. I'll try to make future PR's smaller. The key file is multi_repo_scan.py. 🍏 - a green apple for fun!

### CODE REVIEW

**Comment by jspeed-meyers** [comment]
Had a test that fails in GitHub actions (linux) but passes on local (windows) machines. Huh? Marked that test with xfail for now. Will check on iqt box.

**Comment by jspeed-meyers** [comment]
New theory: (I think) the GitHub action runner is hitting the GitHub API limit and this causes the test to fail. My apple IQT machine has the same results like GitHub actions (fail) without the personal access token but then passes like the windows machine once I add the personal access token. I believe there is a way to add the GitHub personal access token as an encrypted secret to GitHub actions. Let me work on this.

**Comment by jspeed-meyers** [comment]
Revision: I think that we should NOT add the GitHub API token to GitHub actions, making a successful run dependent on the GitHub API token. This is because when other people (I'm being optimistic here) fork our repo and make pull requests, their PR, to pass the test suite, will need to have its own GitHub API token inserted as an encrypted secret, which adds more work to someone submitting a PR. Since I don't want to add work to other potential contributors, I think we can just mark as `xfail` this one test, which I have already done, and consider the test suite adequate. It's not optimal, but clearly this is an unanticipated (from my perspective) downside of making open source software that is dependent on a rate limited API. Live and learn. Also, my $.02, I think the test suite is adequate enough even with this additional test marked as xfail.

File: multi_repo_scan.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+from custom_csv import create_csv, add_committer_to_csv
+from geolocation import get_country_from_location
+from github import get_contributors, get_contributor_location
+from pypi import extract_github_owner_and_repo
+
+
+def scan_multiple_repos(input_file="repos.txt"):
+    """Create csv of data for multiple repos.
+
+    Scan through repo's provided in repos.txt and create a single csv that
```
Comment (kdobolyi) [inline multi_repo_scan.py:14]: typo I think? "repos"

**Comment by kdobolyi** [comment]
left a minor comment about a typo

File: multi_repo_scan.py (hunk 1 above)
Comment (jspeed-meyers) [reply multi_repo_scan.py:14]: Thank you, @kinga! Agreed on the typo.

**Pull request merged**